### ISSUE TITLE
Heatmap Panel - 5 min history for 7 days but data older than this are merged into 1 hour

### ISSUE THREAD

**Issue by Rudorufu-san** [alexanderzobnin/grafana-zabbix#1171]
Greetings,

We found a possible bug.
In Zabbix we have items, that monitors in 5  minutes intervals. We want to display these items in Grafana. For this, we use a plugin Heatmap Panel (Native Plugin). We can see whole 5 min history for 7 days but data older than this are merged into 1 hour intervals despite the fact, that in Zabbix we hold 5 min history for 90 days. Is there a way how to show whole 90 days history not just 7 days in Grafana?

Best regards

**Software versions**
| Grafana | Zabbix | Grafana-Zabbix Plugin |
| 7.4.3   | 5.0.6  | 4.1.3                 |

**Comment by jedd** [comment]
In Grafana, in your DataSource configuration for Zabbix, what setting do you have under the 'Zabbix API details', specifically:

- Trends (on/off)
- After:  (value)
- Range: (value)

If you hover over the 'i' on those last two you'll get a clue as to what they mean / define.

**Comment by Rudorufu-san** [comment]
No. 1 THX this is point! Its work.
